examples/foundational/15-switch-voices.py

```diff
@@ -10,6 +10,7 @@
 from dotenv import load_dotenv
 from loguru import logger
 from openai.types.chat import ChatCompletionToolParam
+from websockets.frames import Frame
```

Comment:
Looks like a typo. Should be:

```suggestion
from pipecat.frames.frames import Frame
```

Reply:
I'd have swore I fixed that. Done!